test/extended/etcd/helpers/helpers.go

```diff
@@ -180,6 +183,91 @@ func recoverClusterToInitialStateIfNeeded(ctx context.Context, t TestingT, machi
 	})
 }
 
+func DeleteSingleMachine(ctx context.Context, t TestingT, machineClient machinev1beta1client.MachineInterface) (string, error) {
+	waitPollInterval := 15 * time.Second
+	waitPollTimeout := 5 * time.Minute
```

Comment:
you probably want to check this every 30s or so.  The load is minimal and it helps latency.  Just fixing in master is ok with me.

Reply:
thanks @deads2k 

This is a backport, maybe @hasbro17  we need a new PR with these changes to 4.13 before backporting ?